build/charts/antrea-windows/conf/ovs/Run-AntreaOVS.ps1

```diff
@@ -22,18 +22,22 @@ ovs-vsctl --no-wait init
 $OVS_VERSION=$(Get-Item $OVSDriverDir\OVSExt.sys).VersionInfo.ProductVersion
 ovs-vsctl --no-wait set Open_vSwitch . ovs_version=$OVS_VERSION
 
-ovs-vswitchd --log-file=/var/log/antrea/openvswitch/ovs-vswitchd.log --pidfile -vfile:info --detach
-
+# Use RetryInterval to reduce the wait time after restarting the OVS process, accelerating process recovery.
```

Comment:
why does it accelerate recovery?
If we have just restarted ovsdb-server, do we think it is necessary to check the ovsdb-server process right away (after 2s), as opposed to waiting for 30s? Do we expect the restart to fail, or is it to make sure that we have a chance to restart ovs-vswitchd quickly?

Reply:
@antoninbas In a recent issue we investigated, after the Windows host rebooted, the ovsdb-server took about 8–20 seconds to become available. During this period, the ovs-vswitchd process failed to start, showing a "pipe ended" error. If we keep the 30-second interval, the ovs-vswitchd process would not recover until at least 30 seconds after the error occurs, resulting in the antrea-agent container crashing multiple times due to ovs-client failures.
To improve this, we added an additional `RetryInterval`. When restarting any OVS process, this reduces the wait time before checking the process status again. Once all processes are confirmed to be available, the check interval reverts to the regular 30 seconds to reduce resource usage.